documentation/src/docs/metrics.md

```diff
@@ -308,58 +368,66 @@ public class MetricsConfig extends ComponentConfig {
 }
 ```
 
-# 4. Steps to add your own Metrics
-* Step 1. When start a segment store/controller service, start a Metrics service as a sub service. Reference above example in ServiceStarter.start()
+# Configuring Own Metrics
+
+1. On the start of a Segment Store/Controller Service, start a Metric Service as a sub service. Please check [`ServiceStarter.start()`](#example-for-starting-a-metric-service)
+
 ```java
 public class AddMetrics {
         statsProvider = MetricsProvider.getProvider();
         statsProvider.start(metricsConfig);    
-    // Step 2. In the class that need Metrics: get StatsLogger through MetricsProvider; then get Metrics from StatsLogger; at last report it at the right place.
+```
+
+2. In the class that need Metrics: get `StatsLogger` through `MetricsProvider`; then get `Metrics` from `StatsLogger`; in the end report it at the right place.
+
```

Comment:
@kevinhan88 can u rephrase the step 2...?!not clear

Reply:
I may suggest something like:
2. Create a new 'StatsLogger' instance through MetricsProvider.createStatsLogger(String loggerName), and register metric by name, e.g. "createCounter(String name)"; and then update the metric object as appropriate somewhere in your code.

Reply:
@kevinhan88 Accepted suggestion